docs/illink-tasks.md

```diff
@@ -53,20 +53,30 @@ The linker can be invoked as an MSBuild task, `ILLink`. We recommend not using t
         ExtraArgs="-t -c link" />
 ```
 
-## Default Linking Behaviour
+## Default Linking Behavior
 
-### .NET 5.0
+The linker default behavior depends on the SDK, because each SDK uses linker settings appropriate for the target form factor.
 
-By default, the linker will operate in a full linking mode for all framework or 
-core managed assemblies. The 3rd party libraries or final app will be analyzed but not linked.
-This setting can be altered by setting `_TrimmerDefaultAction` property to different
-linker action mode.
+### .NET Core SDK
 
-### .NET 3.x
+By default, only framework assemblies are trimmed, and they are trimmed in a conservative
+assembly-level mode (`copyused` action). Third-party libraries and the app will be analyzed but not trimmed.
 
-By default, the linker will operate in a conservative mode that keeps
-all managed assemblies that aren't part of the framework (they are
-kept intact, and the linker simply copies them).
+### Blazor SDK
+
+By default, framework assemblies are trimmed in an aggressive member-level mode (`link` action). ASP.NET assemblies
+use type-level trimming (achieved by using custom root descriptors with the global `TrimMode` `link`), and third-party libraries are analyzed
+but not trimmed.
+
```

Comment:
I'm not sure this is worth mentioning here. It's pretty confusing for seasonal Web Developer and most of the terms are not self-describing. I'd just remove it as we don't need it here